content/docs/explanation/models/data-models-full-example/index.en.md

```diff
@@ -339,7 +339,7 @@ Operational point is also known in French as "Point Remarquable" (PR).
 One `OperationalPoint` is a **collection** of points (`OperationalPointParts`) of interest.
 
 For example, it may be convenient (reference point for train operation) to store the location of platforms as parts and group them by station in operational points.
-In the same way, a bridge over tracks will be **one** OperationalPoint, but it will have **several** OperationPointParts, one at the intersection of each track.
+In the same way, a bridge over tracks will be **one** OperationalPoint, but it will have **several** OperationPointParts, one at the intersection of each track. The `local_track_name` allows users to distinguish OperationPointParts without knowing the underlying track section ids.
```

Comment:
"The \<name\>" doesn't sound right. I'd suggest adding "field": "The local_track_name field". Alternatively, drop "the".